README.md

```diff
@@ -26,36 +26,54 @@ $ npm i -D eslint
 $ yarn add -D eslint
 ```
 
-3. Create a `.eslintrc` file
+3. Create a `.eslintrc.js` file
 
 4. Extend our config (you can use just the scope name as ESLint will assume the `eslint-config` prefix):
 
-```json
-{
-  "extends": [
-    "@nuxtjs"
+```js
```

Comment:
Any objection to prefer `.js`?

Reply:
I don't know, I've been able to see both.
i.e. Http module use it, but PWA module use `.eslintrc`

BUT I just checked and forgot that `.eslintrc` has Code editor support so it fits better
I should rollback these changes

Reply:
(personal opinion) I prefer JSON format as it is simpler and has less boilerplate but js allows more advanced features like custom configuration based on env, etc. IDE support is little more for js (vim, ...) -- just asking to know if there is special typescript requirement. I'm OK with both :)

/cc @manniL @clarkdo

Reply:
Okay, same I'm ok with both, so I reverted so that this PR is lighter, we'll maybe update README afterwards